ui/app/components/ens-input.js

```diff
@@ -6,7 +6,7 @@ const debounce = require('debounce')
 const copyToClipboard = require('copy-to-clipboard')
 const ENS = require('ethjs-ens')
 const networkMap = require('ethjs-ens/lib/network-map.json')
-const ensRE = /.+\.eth$/
+const ensRE = /.+\..+$/
```

Comment:
regex is so unreadable - i have no idea what this does

Reply:
It just makes sure there's a period between some other things. It's barely anything.

To break it down:
`/`: Begin regex.
`.+`: Allow anything for a while
`\.`: Require a period
`.+`: Allow anything for a while
`$`: Before the end
`/`: End regex.

Reply:
how about `string.includes('.')`?